site/src/css/custom.css

```
@@ -279,6 +279,12 @@ div.text--center {
   font-size: 8px;
 }
 
+/* Change vertical alignment of operations to align with summary*/
+.RedocStandalone div.menu-content div.scrollbar-container 
+  ul li label span.operation-type {
+  margin-top: 4px;
```

```suggestion
  margin-block: auto;
```
I didn't test in all browsers but if that works I like it better than the magic number. Also, the `4px` didn't align exactly on my machine, it was better though. 🥈